pages/History.qml

```diff
@@ -633,7 +633,8 @@ Rectangle {
                                 MoneroComponents.TextPlain {
                                     font.family: MoneroComponents.Style.fontRegular.name
                                     font.pixelSize: 15
-                                    text: (isout ? qsTr("Sent") : qsTr("Received")) + translationManager.emptyString
+                                    text: (isout ? qsTr("Sent") + (isFailed ? "<style type='text/css'>.red {color:#ff0000;}</style>" + " (" + qsTr("Failed") + ") " + "<span class='red'>" + FontAwesome.timesCircle + "</span>" : (isPending ? " (" + qsTr("Pending") + ")" : "")) : qsTr("Received")) + translationManager.emptyString
+                                    textFormat: Text.RichText
```

Comment:
Could you post a screenshot of a failed tx?

Reply:
![image](https://user-images.githubusercontent.com/45968869/100940943-21975900-34f9-11eb-93dc-951b1c434ba9.png)

Reply:
IMO `ban` icon might look better here. What do you think?

Reply:
For some reason FontAwesome.ban (and some other FontAwesome icons) are not being displayed, a rectangle with a X inside is displayed instead:
![image](https://user-images.githubusercontent.com/45968869/101097846-3855b280-35c2-11eb-96dc-1ff66fba6712.png)

Reply:
Did you set font.styleName: "Solid" ?

Reply:
It did the trick... I had forgot to set the family and style.
Some options below (I prefer using ban or times icon, and I prefer using the icon on the left side)

ban:
![image](https://user-images.githubusercontent.com/45968869/101100076-27a73b80-35c6-11eb-8b51-8926e026edae.png)

timesCircle:
![image](https://user-images.githubusercontent.com/45968869/101100601-16126380-35c7-11eb-9384-ff0c20fc1742.png)

exclamationTriangle:
![image](https://user-images.githubusercontent.com/45968869/101100215-64733280-35c6-11eb-9276-82a642a1f4d8.png)

exclamationCircle:
![image](https://user-images.githubusercontent.com/45968869/101100331-9c7a7580-35c6-11eb-9f48-9da27d32f4ac.png)

exclamation:
![image](https://user-images.githubusercontent.com/45968869/101100452-d21f5e80-35c6-11eb-89b0-1d85ef795862.png)

times: 
![image](https://user-images.githubusercontent.com/45968869/101100571-04c95700-35c7-11eb-9982-9ec42f3c6028.png)

Alternative (display status icon on left side):

![image](https://user-images.githubusercontent.com/45968869/101100832-83be8f80-35c7-11eb-8096-3e1d908bcec4.png)

Reply:
> Alternative (display status icon on left side):

This one looks good to me. What do you guys think?

Reply:
Agree.

Reply:
Updated. I also added a clock symbol to pending transactions (with 0 confirmations):
![image](https://user-images.githubusercontent.com/45968869/101144460-a75df600-3618-11eb-9e0d-ab427c93c5ec.png)